GuessMyNumber/Readme.md

```diff
@@ -0,0 +1,131 @@
+# Guess My Number
+
+Let's implement a classic "Guess My Number" game. The program generates a random number between 0-99 and prompts if your guess is correct, too high, or too low.
+
+### Generate the secret number
+
+```lean
+import Std.Data
+
+-- Generate a "random" number in [0, 99]
+-- It's not actually random. We'll talk about it later.
+def GetSecret : IO Nat :=
+  IO.rand 0 99
+```
+
+First, `IO.rand` is not actually a random number. It's a pseudo random number generator **with the same seed every time**! But that's not important for this discussios. Let's assume it returns a number randomly.
```

Comment:
@leodemoura We should probably move this to the `IO.stdGenRef` initializer https://github.com/leanprover/lean4/blob/8649483b416ec9d919c380d657ceedf8184ed527/src/Lean/Server/FileWorker.lean#L494-L495

Reply:
Fixed, thanks.